SB2018/2018-04-12.md

```diff
@@ -0,0 +1,52 @@
+---
+to: All Teams
+subject: Rules amendments
+---
+
+Hi!
+
+Some issues have come to light with [our rules][rules], and we've made some amendments:
+
+## Amendment 1: Token Marker ID changed
+
+The first amendment changes the IDs of marker tokens, as defined in section 3.1. This is of relevance to your team if they have hard-coded the IDs of the tokens from the rules. There's no need to update your robots to have the new numbers, they are already correct.
+
+The old marker numbers were as follows:
+
+| Item                                    | Marker Number     | Marker Size      |
+|-----------------------------------------|-------------------|------------------|
+| Arena boundary                          |     0 -- 27       |           250    |
+| Columns                                 |    28 -- 43       |           250    |
+| Tokens belonging to the robot in zone 0 |    **44 -- 49**   |           100    |
+| Tokens belonging to the robot in zone 1 |    **50 -- 54**   |           100    |
+| Tokens belonging to the robot in zone 2 |    **55 -- 59**   |           100    |
+| Tokens belonging to the robot in zone 3 |    **60 -- 63**   |           100    |
+
+The amended marker numbers are as follows:
+
+| Item                                    | Marker Number     | Marker Size      |
+|-----------------------------------------|-------------------|------------------|
+| Arena boundary                          |     0 -- 27       |           250    |
+| Columns                                 |    28 -- 43       |           250    |
+| Tokens belonging to the robot in zone 0 |    **44 -- 48**   |           100    |
+| Tokens belonging to the robot in zone 1 |    **49 -- 53**   |           100    |
+| Tokens belonging to the robot in zone 2 |    **54 -- 58**   |           100    |
+| Tokens belonging to the robot in zone 3 |    **59 -- 63**   |           100    |
+
+## Amendment 2: Scoring area sizes
+
+The second amendment fixes some ambiguity on the size of the scoring zones, as defined in section 3.2.11. This is of relevance to your team if they use the measurements of the arena to define the movement of the robot.
+
+Previously, the specification in Section 3.2.11 did not match the arena layout in Figure 2, the specification states that the scoring zones were squares of length 3m. When reading the figure, one could derive that the scoring zones are squares of 2.815m. The amendment changes the definition in the specification to match that of the diagram.
+
+The old specification stated:
+
+> 11. The scoring zones are squares of sides *3m* positioned with the columns separating
+them
+
+The amended specification states:a
```

Comment:
Is the `a` after the colon on this line intentional?

Reply:
fixed in 
159eb15